packages/unlock-js/src/walletService.ts

```diff
@@ -167,7 +167,7 @@ export default class WalletService extends UnlockService {
       erc20Address?: string
       decimals?: number
       recurringPayments?: number
-      referer?: string
```

Comment:
That typo was intentional because https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer :) but it;s ok to clean i as long as you check that it does not affect the rest of the app?